### DISCUSSION TITLE
Choose caller id in warm and cold transfer

### DISCUSSION THREAD

**Discussion by MDavidTorres** [twilio-professional-services/flex-project-template#637] [Q&A]
Good afternoon, colleagues.

I need to know how to choose the caller ID when making a warm transfer and a cold transfer to an external Twilio number. At the moment it seems to be random, but I need to be able to choose from which number or at least for it to always be the same.

**Comment by dremin** [comment]
Hi @MDavidTorres! The caller ID selected in the outbound dialing panel (when clicking the dialpad icon in the main navigation header) will be used as the caller ID for transfers as well.

**Reply by MDavidTorres** [reply]
is it an issue then? Because I have done several tests where the caller id is different

**Reply by dremin** [reply]
Are you seeing the caller ID not match the selected one? If so, does your copy of the template include the changes from #348?
